hq/test/aws/ec2/EC2Test.scala

```diff
@@ -140,6 +142,38 @@ class EC2Test extends FreeSpec with Matchers with Checkers with PropertyChecks
     }
   }
 
+  "VPC" - {
```

Comment:
The formatting's a bit off in the block. Pay attention to unneeded empty lines, indentation and whitespace in signatures.